tests/e2e/utils/features.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Enables or disables a feature flag in E2E tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { wpApiFetch } from './wp-api-fetch';
+
+/**
+ * Disables a feature flag during E2E tests.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} feature The feature flag to disable.
+ * @return {Promise} Promise from `page.evaluate()` call.
+ */
+export async function disableFeature( feature ) {
+	try {
+		// Wait until apiFetch is available on the client.
+		await page.waitForFunction( () => window.fetch !== undefined );
+
+		return await wpApiFetch( {
+			path: 'google-site-kit/v1/e2e/feature/set-flag',
+			method: 'post',
+			data: { [ feature ]: false },
+		} );
+	} catch ( error ) {
+		// eslint-disable-next-line no-console
+		console.warn( 'disableFeature failure', page.url(), feature );
+		throw error;
+	}
```

Comment:
This can be simplified a good deal I think.

* the `waitForFunction` is unnecessary here as this is already done in the `wpApiFetch` utility.
* The catch block here with the console log also happens in `wpApiFetch` so this would result in multiple logs for one error which isn't really necessary. We should have enough information from the `wpApiFetch` failure to know it came from here.

Reply:
Ah, shoot, okay. This technique is copied from another, similar E2E util. I guess there are a few places where the code is overly complicated/boilerplate-y. I'll tidy this one up, at least.